theme/themes/pastanaga/extras/grid.less

```diff
@@ -0,0 +1,405 @@
+// Row block
+@blueArctic: #e2f1fd;
+@greySnow: #f3f5f7;
+@greySmoke: #e4e8ec;
+@grid-images-aspect-ratio: var(--grid-images-aspect-ratio, 16/9);
+@grid-images-object-position: var(--grid-images-object-position, top left);
+
+.block-editor-gridBlock {
+  position: relative;
+  // Removed because it broke the z-index of the blockChooser, making it fall behind
+  // z-index: 10;
+
+  .block {
+    height: 100%;
+
+    .image {
+      & > div {
+        height: 100%;
+      }
+
+      & > div > div {
+        height: 100%;
+      }
+    }
+  }
+}
+
+// Override handler TODO: do it better (only for grids)
+body.has-toolbar.has-sidebar .drag.gridBlock.handle.wrapper {
+  // TODO: review
+  margin-top: 5px;
+  margin-left: 0px !important;
+}
+
+// Inner Remove block button
+.ui.basic.button.remove-block-button {
+  position: absolute;
+  z-index: 3;
+  top: 6px;
+  right: 2px;
+  display: none;
+
+  .icon {
+    height: 18px !important;
+  }
+
+  .circled {
+    padding: 8px;
+    border: 1px solid @greySmoke;
+    background: @greySnow;
+
+    &:hover {
+      background-color: @greySmoke;
+    }
+  }
+
+  &:hover {
+    background-color: transparent !important;
+  }
+}
+
+.block.gridBlock,
+.block.column {
+  [class^='block-editor-'].selected .ui.basic.button.remove-block-button,
+  [class^='block-editor-']:hover .ui.basic.button.remove-block-button {
+    display: block;
+  }
+}
+
+// The Empty block cell (yet to be added)
+.gridBlock-empty-placeholder {
+  position: relative;
+  display: flex;
+  height: 100%;
+  min-height: 200px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  background: @blueArctic;
+
+  &.selected::before {
+    position: absolute;
+    // otherwise, this layer is above the object, then it does not allow clicking inside
+    z-index: -1;
+    top: -5px;
+    left: -5px;
+    width: calc(100% + 10px);
+    height: calc(100% + 10px);
+    border: 1px solid rgba(120, 192, 215, 0.5);
+    border-radius: 3px;
+    content: '';
+  }
+
+  .ui.basic.button.add-block-button {
+    box-shadow: 0px 0px 0px @2px transparent inset !important;
+
+    .icon {
+      height: 32px !important;
+    }
+
+    .circled {
+      padding: 10px;
+      border: none;
+      background: @greySnow;
+
+      &:hover {
+        background-color: @greySmoke;
+      }
+    }
+
+    &:hover {
+      background-color: transparent !important;
+    }
+  }
+
+  .blocks-chooser {
+    top: 0;
+    left: 50%;
+    transform: translate(-50%, 0);
+  }
+}
+
+// Required to overcome the droppable from b-d&d at the bottom
+// !! Required for ALL blocks that have b-d&d and a grid
+.block.gridBlock,
+[class$='Row'].block {
```

Comment:
Are the selectors for `[class$='Row']` still used?

Reply:
Removed